include/picongpu/version.hpp

```diff
@@ -22,4 +22,4 @@
 #define PICONGPU_VERSION_MAJOR 0
 #define PICONGPU_VERSION_MINOR 5
 #define PICONGPU_VERSION_PATCH 0
-#define PICONGPU_VERSION_LABEL "dev"
+#define PICONGPU_VERSION_LABEL ""
```

Comment:
should this not be "dev"?

Reply:
right this should not be part of this PR